nifi-nar-bundles/nifi-influxdb-bundle/nifi-influxdb-processors/src/test/java/org/apache/nifi/processors/influxdb/ITExecuteInfluxDBQuery.java

```diff
@@ -110,8 +125,72 @@ public void testCreateDB() {
         assertEquals("Value should be equal",null, flowFiles.get(0).getAttribute(ExecuteInfluxDBQuery.INFLUX_DB_ERROR_MESSAGE));
         assertEquals("Value should be equal",query, flowFiles.get(0).getAttribute(ExecuteInfluxDBQuery.INFLUX_DB_EXECUTED_QUERY));
 
-        flowFiles.get(0).assertContentEquals("{\"results\":[{}]}");
+        QueryResult queryResult = gson.fromJson(new StringReader(new String(flowFiles.get(0).toByteArray())), QueryResult.class);
+        assertEquals("results array should be empty", 1, queryResult.getResults().size());
+        assertEquals("No series", null, queryResult.getResults().get(0).getSeries());
+    }
+
+    @Test
+    public void testEmptyFlowFileQueryWithScheduledQuery() {
+        String message = "water,country=US,city=newark rain=1,humidity=0.6 1501002274856668652";
+        influxDB.write(dbName, DEFAULT_RETENTION_POLICY, InfluxDB.ConsistencyLevel.ONE, message);
+
+        String query = "select * from water";
+        runner.setProperty(ExecuteInfluxDBQuery.INFLUX_DB_QUERY, query);
+
+        byte [] bytes = new byte [] {};
```

Comment:
Redundant because you can use `runner.enqueue("")` for cases like this where you're not examining the original input flowfile contents.